source/common/ssl/context_impl.cc

```diff
@@ -146,6 +146,16 @@ ContextImpl::ContextImpl(ContextManagerImpl& parent, Stats::Scope& scope,
 
   if (verify_mode != SSL_VERIFY_NONE) {
     SSL_CTX_set_verify(ctx_.get(), verify_mode, nullptr);
+
+    // NOTE: We're using SSL_CTX_set_cert_verify_callback() instead of X509_verify_cert()
```

Comment:
Looks like you moved this out from the `if (!caCert.empty())` block in the merge commit? Any reason to do that? Like I said previously, it's useless there, since `X509_verify_cert()` won't be called without trusted CA.

Reply:
Ok, I'll move it back.  I was trying to put both verify callback configurations in one place.  It seems equivalent since `if (!caCert.empty())` sets `verify_trusted_ca_` and `verify_mode`.  After the merge, `verify_trusted_ca_` was available.